### ISSUE TITLE
Upgrade to new jar versions.

### ISSUE THREAD

**Issue by bgeorge111** [marklogic-community/kafka-marklogic-connector#51]
The connector is using older versions of data hub and client api jars. Also, the kafka connect-api and connect-json jars are also old by view versions.  The ticket is to build a newer connector version with new versions of dependent jars. 

While locally working on newer versions against latest confluent kafka distribution, faced a problem of jackson version mismatch.   
The solution was to exclude jackson modules in the connector jar. This aspect also might need to looked into. All kafka distributions might not have these jars provided during run time.

**Comment by rjrudin** [comment]
Are you looking for the latest version of the Kafka connect-api - i.e. 2.8.0?

**Comment by bgeorge111** [comment]
Yes.

**Comment by rjrudin** [comment]
@bgeorge111 I just did a test of the connector on both Kafka 2.8.1 and Kafka 3.2.0, and it worked fine - i.e. documents were successfully written. The connector does not include any Kafka classes. I noticed too that Mongo's connector is [also compiled on an earlier version - 2.6.0](https://github.com/mongodb/mongo-kafka/blob/master/build.gradle.kts#L53) and I'm assuming it works fine as well on more recent versions. 

That said, we are going to do a new release soon that bumps up every dependency, including the version of Gradle that's used for building the connector. So I'm going to close this ticket as that release will cover the Kafka compile-only dependencies and every other dependency too.
